README.md

```diff
@@ -28,4 +30,6 @@ Application reads configuration from following environment variables
 | Name       | Description                         |
 | ---------- | ----------------------------------- |
 | PORT       | HTTP port to listen to              |
-| ZOOKEEPERS | Comma separated list of Zookeepers  |
+| KAFKAHOSTS | Comma separated list of kafka broker hosts  |
+| TOPICS     | Comma separated list of topics to scrape |
+| CONSUMERGROUPS | a json string specifying the topic and the list of consumer groups |
```

Comment:
Add a bit more detailed explanation about the schema (e.g. map of topic name to array of consumer groups or something similar).